doc/source/make_tables.py

```diff
@@ -406,7 +406,8 @@ def get_row(cls, i, row_data):
 
 def _get_doc(func: Callable[[], Any]) -> Optional[str]:
     """Return the first line of the callable's docstring."""
-    return doc.splitlines()[0] if (doc := func.__doc__) else None
+    doc = func.__doc__
+    return doc.splitlines()[0] if (doc) else None
```

Comment:
Parentheses were only necessary for the assignment expression:
```suggestion
    return doc.splitlines()[0] if doc else None
```